Alignment/OfflineValidation/python/TkAlAllInOneTool/GCP_TkAlMap.py

```diff
@@ -0,0 +1,96 @@
+    print(' - color palette                    : '+str(palette))
+    print(' - input root file                  : '+in_file)
+    print(' - output directory                 : '+out_dir)
+    print(' - saving as png                    : '+str(save_png))
+    print(' - saving as pdf                    : '+str(save_pdf))
+    print('')
+    print('Active plots:')
+    print(' - plot 4 sigma capped values       : '+str(do_4scap))
+    print(' - plot default range capped values : '+str(do_drange))
+    print(' - plot un-capped values            : '+str(do_frange))
+    print(' - plot full detector               : '+str(do_full))
+    print(' - plot pixel detector              : '+str(do_pixel))
+    print(' - plot strips detector             : '+str(do_strips))
+    print('')
+    print('Changed default ranges:')
+    for var in var_ranges:
+        if var_ranges[var][0] is None and var_ranges[var][1] is None: continue
+        prt_srt = ' - '+var+'\t: [ '
+        if var_ranges[var][0] is None: prt_srt += 'default'
+        else: prt_srt += str(var_ranges[var][0])
+        prt_srt += '\t, '
+        if var_ranges[var][1] is None: prt_srt += 'default'
+        else: prt_srt += str(var_ranges[var][1])
+        prt_srt += '\t]'
+        print(prt_srt)
+
+    print('Loading maps')
+    TkMap_full          = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=False, GEO_file=geometry_file, tracker='full',   palette=palette, check_tracker=auto_tk)
+    TkMap_pixel         = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=False, GEO_file=geometry_file, tracker='pixel',  palette=palette, check_tracker=auto_tk)
+    TkMap_strips        = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=False, GEO_file=geometry_file, tracker='strips', palette=palette, check_tracker=auto_tk)
+    TkMap_cap_full      = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=True,  GEO_file=geometry_file, tracker='full',   palette=palette, check_tracker=auto_tk)
+    TkMap_cap_pixel     = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=True,  GEO_file=geometry_file, tracker='pixel',  palette=palette, check_tracker=auto_tk)
+    TkMap_cap_strips    = TkAlMap('test', title, in_file, use_default_range=False, two_sigma_cap=True,  GEO_file=geometry_file, tracker='strips', palette=palette, check_tracker=auto_tk)
+    TkMap_drange_full   = TkAlMap('test', title, in_file, use_default_range=True,  two_sigma_cap=False, GEO_file=geometry_file, tracker='full',   palette=palette, check_tracker=auto_tk)
+    TkMap_drange_pixel  = TkAlMap('test', title, in_file, use_default_range=True,  two_sigma_cap=False, GEO_file=geometry_file, tracker='pixel',  palette=palette, check_tracker=auto_tk)
+    TkMap_drange_strips = TkAlMap('test', title, in_file, use_default_range=True,  two_sigma_cap=False, GEO_file=geometry_file, tracker='strips', palette=palette, check_tracker=auto_tk)
+
```

Comment:
please remove empty lines at the end of this file.

Reply:
Done